### PR TITLE
[Elementor] Translate "price" from Price list widget

### PR DESCRIPTION
**Pull request by andrescifuentesr**
Making the Price field translatable for Price list widget
https://onthegosystems.myjetbrains.com/youtrack/issue/compsupp-6636

### CODE REVIEW

**Comment by dgwatkins** [comment]
@andrescifuentesr I don't think we shoud make the price translatable for everyone:

- ATE doesn't know how to translate a price
- Most customers wont want to translate prices anyway.

If a certain customer needs, he can set it in his local XML.

What do you think?

**Comment by andrescifuentesr** [comment]
@dgwatkins I had the same doubts about setting this field to translate or not. The initial request was done because the user was not adding just a number but also currencies and weight, ie 5€/kg.
So, if you just add a number, that would be OK as the value will be copied directly, however, if you add a currency, you may want to translate it.
Anyway, we can leave it as it is and collect more reports, if it makes sense.

**Comment by dgwatkins** [comment]
Thanks @andrescifuentesr 

I will close here then and we can follow up on the ticket.

**Comment by dgwatkins** [comment]
Oops, sorry @andrescifuentesr 

I realize now that this PR has more than just the price.

I think you should remove the price from this pull request and then we can merge.
Makes sense?

File: elementor/wpml-config.xml [hunk 1]
```diff
@@ -337,5 +337,13 @@
                 <field type="Countdown: message after expire">message_after_expire</field>
             </fields>
         </widget>
+        <widget name="price-list">
+            <fields-in-item items_of="price_list">
+                <field type="Pricing list: title">title</field>
+                <field type="Pricing list: description">item_description</field>
+                <field type="Pricing list: price">price</field>
```
Comment (dgwatkins) [inline elementor/wpml-config.xml:344]: Remove the above line @andrescifuentesr

File: elementor/wpml-config.xml [hunk 2]
```diff
@@ -337,5 +337,13 @@
...
+                <field type="Pricing list: link">link>url</field>
```
Comment (dgwatkins) [inline elementor/wpml-config.xml:345]: And lets add the editor_type here:
```suggestion
                <field type="Pricing list: link"editor_type="LINK">link>url</field>        
```

**Review by dgwatkins — CHANGES_REQUESTED** [review]

**Comment by andrescifuentesr** [comment]
Thanks for the help, David.
I think we are good to go.

**Review by dgwatkins — APPROVED** [review]

**Comment by dgwatkins** [comment]
fixed,  merged and deployed @andrescifuentesr

**Pull request merged**